### DISCUSSION TITLE
Automatic change of base branch broken if branch is deleted from the android app

### DISCUSSION THREAD

**Discussion by HendrikKlug-synthara** [community/community#127281] [Mobile]
### Select Topic Area

Bug

### Body

The automatic change of the base branch of a pull request to main when the base branch gets deleted does not work if the branch gets deleted from the phone github app.
If the branch gets deleted from the android app, instead of changing the base branch of the pull request to main, the pull request gets closed.
See this pull request for example: https://github.com/HendrikKlug-synthara/mytest/pull/227
The pr was closed and the branch was deleted from the android app.
The next pr that was targeting that branch was then closed instead of having the base branch changed to main: https://github.com/HendrikKlug-synthara/mytest/pull/228
If I instead delete the branch from the web interface, the base branch of the pr gets changed to main as expected.
Let me know if I'm doing something wrong. Thanks!

**Comment by esahmed21** [comment]
Hi @HendrikKlug-synthara 👋 

Thank you for sharing the steps in details, I was able to reproduce it on the iOS client as well. I will create internal issue for the team to track this bug and fix it!

Thank you for bringing this to our attention!

**Reply by HendrikKlug-synthara** [reply]
Good to know, thanks for the reply!

**Comment by HendrikKlug-synthara** [comment]
I have the same issue when deleting the remote branch with `git push origin --delete {mybranch}` which is more annoying since that means that I cannot do a fast forward merge of a pull request and delete the branch from the cli...
